_flake8_tergeo/checks/ast_func_def.py

```diff
@@ -256,3 +257,35 @@ def _check_valid_return_annotation(node: AnyFunctionDef) -> IssueGenerator:
             issue_number="107",
             message="Instead of using typing.Never for return annotations, use typing.NoReturn.",
         )
+
+
+def _check_override_first_decorator(node: AnyFunctionDef) -> IssueGenerator:
+    decorator_list = node.decorator_list
+    if not decorator_list or len(decorator_list) == 1:
+        return
+
+    for index, decorator in enumerate(decorator_list):
+        if not is_expected_node(decorator, "typing", "override"):
```

Comment:
What about typing_extensions?

Reply:
is_expected_node does automatically check for typing_extensions as well